### Task 1
Два економісти заповнюють документи з власними прогнозами щодо курсу певних коштовних паперів, після чого їх складають у спільну папку. Ймовірність відхилення прогнозованого значення курсу від фактичного понад допустиму похибку для першого економіста дорівнює 0,1; для другого 0,2. Перший економіст заповнив 40 документів, другий - 60. Під час перевірки навмання взятий із папки документ виявився з суттєвою похибкою. Визначити ймовірність того, що його склав перший економіст.




Враховуючи ймовірність похибки у кожного з економістів, загальна ймовірність хибних прогнозів буде складати:  
Р(А) = 0,4 х 0,1 + 0,6 х 0,2  
Р(А) = 0,16  
Згідно формули Байеса, ймовірність того що хибний прогноз склав 1й економіст буде дорівнювати:  
Р(А1|A) = (0,4 x 0,1) / 0,16  
Р(А1|A) = 1/4 = **0,25**

### Task 2
Є можливість вибрати спосіб виробництва і реалізації двох наборів товарів широкого вжитку. За даними відділу маркетингу, яким були проведені дослідження ринку, можливий прибуток від виробництва і реалізації Хі У наведено в таблицях (X, Y - прибуток у грошових одиницях):  

X  |1000| 1500| 2000|
:-:|---:|---: |---: |
P  | 0,5| 0,3 |0,2  |


Y  |1000| 1500| 1750|
:-:|---:|---: |---: |
P  | 0,4| 0,4 |0,2  |

Потрібно оцінити ступінь ризику і запропонувати рішення щодо випуску і реалізації одного із наборів товарів.

In [14]:
import numpy as np

In [15]:
x = np.array([1000, 1500, 2000])
p = np.array([0.5 ,0.3, 0.2])
M = sum(x*p)           # Математичне очікування
D = sum(x**2*p) - M**2 # Дисперсія
s = D**0.5             # Стандартне відхилення
print(f' Математичне очікування прибутку від товару Х дорівнює: {M}, Стандарте відхилення: {round(s, 2)}')

 Математичне очікування прибутку від товару Х дорівнює: 1350.0, Стандарте відхилення: 390.51


In [16]:
y = np.array([1000, 1500, 1750])
p = np.array([0.4, 0.4, 0.2])
M = sum(y*p)           # Математичне очікування
D = sum(y**2*p) - M**2 # Дисперсія
s = D**0.5             # Стандартне відхилення  
print(f' Математичне очікування прибутку від товару Y дорівнює: {M}, Стандарте відхилення: {round(s, 2)}')

 Математичне очікування прибутку від товару Y дорівнює: 1350.0, Стандарте відхилення: 300.0


Очікуваний прибуток для товарів Х і У (мат. очікування) є однаковим: **1350** грошових одиниць.   
Але стандартне відхилення для товару Y складає **300** і є меншим ніж для товару Х, яке дорівнює **390.51**    
Тому можно зробити висновок, що виробництово товару **Y** дасть більш стабільний прибуток, тобто є менш ризиковим.